package.json

```diff
@@ -24,7 +24,7 @@
     "tap": "^1.3.2"
   },
   "dependencies": {
-    "nan": "^2.0.5"
+    "nan": "^2.3.5"
```

Comment:
I am not well versed in semver, and double checking the docs confirms that ^2.0.5 matches 2.3.5. However, trying to build on node v6.2.1 and npm 3.9.6, I get the errors shown in #14. Upgrading nan to 2.3.5 in the package.json fixes the issue. Is there something else I am missing?